packages/data/.gitignore

```diff
@@ -30,3 +30,4 @@ CHANGELOG.md
 README.md
 LICENSE
 build/
+/lib/service/retrofit_api_service.g.dart
```

Comment:
avoid specifying such concrete paths. better variant - ignore file retrofit_api_service.g.dart anywhere. the best - ignore  all .g.dart files anywhere

Reply:
fixed